src/coins.h

```diff
@@ -292,12 +292,19 @@ class CCoinsViewCache : public CCoinsViewBacked
         const uint64_t nBestCoinHeight,
         size_t &nChildCachedCoinsUsage);
 
+    /**
+     * Check if we have the given utxo on disk and load it into cache.
+     * The semantics are the same as HaveCoin(), but no calls to
+     * the backing CCoinsView are made.
+     */
+    bool GetCoinFromDB(const COutPoint &outpoint) const;
+
     /**
      * Check if we have the given utxo already loaded in this cache.
      * The semantics are the same as HaveCoin(), but no calls to
      * the backing CCoinsView are made.
      */
```

Comment:
The new fSpent parameter needs to be documented.  Please mention that its only set if the function returns true.

Reply:
done